packages/jest-runner/tsconfig.json

```diff
@@ -17,6 +17,7 @@
     {"path": "../jest-test-result"},
     {"path": "../jest-types"},
     {"path": "../jest-worker"},
-    {"path": "../jest-util"}
+    {"path": "../jest-util"},
+    {"path": "../jest-transform"}
```

Comment:
alphabetical, please 😀

Reply:
Sure, will apply in the other PR as well